site/docs/components/icons.md

```diff
@@ -5,18 +5,23 @@ description: Use SVG Icons in HTML
 group: components
 toc: true
 ---
-# NOT Updated for FLO-SCSS yet
 
-TODO:
- - explain how svgs should not have set width or height so the parent container can set its dimensions
- - explain how it shold not have inline styles or colors, so colors can be changed through CSS and added classes
- - mention logos but link them to the logo page
- - mention badges but link them to the badge page
+We use svgs in the design system and not a font library to display icons.  SVG's are added to a main sprite sheet allowing them to load once. The `<use>` element is how to call an svg from the sprite to the page.  
 
```

Comment:
I would clarify we are only doing this for FLO-SCSS documentation, or not have that at all, until we decide how we will add SVG's to the webapp. Currently we are not using a sprite sheet in the web app

Reply:
Ok